packages/SwingSet/docs/vat-upgrade.md

```diff
@@ -94,32 +100,33 @@ To enable upgrade, the v1 code must stash important data in durable storage, and
 
 For upgrade-important in-RAM objects, the v1 code needs to store a "reattachment handle" (name and API TBD) in durable storage. This gives the v2 code the authority to take over the identity of these exported objects.
```

Comment:
Let's remove this sentence about "reattachment handles", I had originally thought that we'd provide a way to reattach ephemeral singletons (other than the root object), but I don't see a place for that in our new Exo world.